app/src/main/java/org/piramalswasthya/sakhi/helpers/dynamicMapper/FormSubmitRequestMapper.kt

```
@@ -0,0 +1,50 @@
+            null
+        }
+    }
+
+
+    private fun convertDateToIso(input: String): String {
+        return try {
+            val inputFormat = SimpleDateFormat("dd-MM-yyyy", Locale.getDefault())
+            val outputFormat = SimpleDateFormat("yyyy-MM-dd", Locale.getDefault())
+            val date = inputFormat.parse(input)
+            outputFormat.format(date!!)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)